pkg/knuu/instance.go

```diff
@@ -326,6 +343,25 @@ func (i *Instance) AddFile(src string, dest string, chown string) error {
 		}
 		file := k8s.NewFile(dstPath, dest)
 
+		parts := strings.Split(chown, ":")
+
+		if len(parts) != 2 {
+			return fmt.Errorf("invalid format")
+		}
+
+		group, err := strconv.ParseInt(parts[1], 10, 64)
+		if err != nil {
+			return fmt.Errorf("failed to convert to int64: %s", err)
+		}
```

Comment:
We need to also provide more context for this statement. 
From outside context, this might look like a set of magic numbers